kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/editor/documentation_urls.properties

```diff
@@ -0,0 +1,13 @@
+javadoc.home=https://openjfx.io/javadoc/16/
+oracle.documentation=https://docs.oracle.com/javase/8/javase-clienttechnologies.htm
+
+gluon.javadoc.home=https://docs.gluonhq.com/charm/javadoc/latest/
+gluon.scenebuilder.home=https://gluonhq.com/products/scene-builder/
+gluon.scenebuilder.contribute=https://github.com/gluonhq/scenebuilder
+
+openjfx.getting.started=https://openjfx.io/openjfx-docs/
+openjfx.javadoc.home=https://openjfx.io/javadoc/16/
+openjfx.css.reference=https://openjfx.io/javadoc/16/javafx.graphics/javafx/scene/doc-files/cssref.html
+openjfx.fxml.reference=https://openjfx.io/javadoc/16/javafx.fxml/javafx/fxml/doc-files/introduction_to_fxml.html
+
+tornadofx.getting.started.kotlin=https://docs.tornadofx.io/
```

Comment:
A late call but can we update all the reference URLs to 17?

Reply:
All JavaFX links point to version 17 as of now.